torch/_inductor/scheduler.py

```diff
@@ -1519,6 +1551,25 @@ def can_fuse(self, node1: BaseSchedulerNode, node2: BaseSchedulerNode):
         if node2.get_names() & node1.ancestors:
             return False  # node1 must go before node2
 
+        if (
+            isinstance(node1, (FusedSchedulerNode, SchedulerNode))
+            and isinstance(node2, SchedulerNode)
+            and isinstance(node2._body, ir.LoopBody)
```

Comment:
@jansel do we really want to be doing pattern matches on scheduler nodes like this?

Reply:
There was some discussion on the prior PR.  I think including it in MemoryDeps is a bit cleaner... though both methods work.